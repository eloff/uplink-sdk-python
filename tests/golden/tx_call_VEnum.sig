r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 58506979028178122487890331105628780885127379681854076171025270234451953175442
