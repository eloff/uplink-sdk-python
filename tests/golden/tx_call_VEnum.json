{
    "header": {
        "contents": {
            "contents": {
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
                "args": [
                    {
                        "contents": "Foo", 
                        "tag": "VEnum"
                    }
                ], 
                "method": "get"
            }, 
            "tag": "Call"
        }, 
        "tag": "TxContract"
    }, 
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE01ODUwNjk3OTAyODE3ODEyMjQ4Nzg5MDMzMTEwNTYyODc4MDg4NTEyNzM3OTY4MTg1NDA3NjE3MTAyNTI3MDIzNDQ1MTk1MzE3NTQ0Mg=="
}